crates/rspack_core/src/stats/struct.rs

```diff
@@ -220,17 +220,17 @@ pub struct StatsChunk<'a> {
 #[derive(Debug)]
 pub struct StatsChunkGroupAsset {
   pub name: String,
-  pub size: f64,
+  pub size: usize,
```

Comment:
because of rust 1.81.0 the empty list sums will return -0.0

https://github.com/rust-lang/rust/pull/129321